rpc/master/service_client.go

```diff
@@ -109,3 +110,15 @@ func (c *Client) ClearEmergency(serviceID string) (int, error) {
 	err := c.call("ClearEmergency", serviceID, &affected)
 	return affected, err
 }
+
+// Remove the IP assignment of a service's endpoints
+func (c *Client) RemoveIPs(args []string) (err error) {
+	err = c.call("RemoveIPs", args, new(string))
```

Comment:
Please change these methods to "return c.call("RemoveIPs", args, new(string))" The implicit return of the error is more verbose in this case.  Same is true for the other methods doing this.